### DISCUSSION TITLE
SET pg_trgm.similarity_threshold in rpg function or in js supabase query

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#12110] [Questions] [answered]
Im able to SET pg_trgm.similarity_threshold through the sql editor prior to query with expected result.

Struggling to do the same within a rpg function or within a js supabase query. Doable?

Or is there a way to set it globally?

**Comment by steve-chavez** [comment, marked as answer]
> Or is there a way to set it globally?

Yes, you can set it as:

```sql
ALTER ROLE authenticator SET pg_trgm.similarity_threshold..
```

This will affect all queries made through `supabase-js`.

Also, make sure to do `NOTIFY pgrst, 'reload schema'` to reload all db connections so they pick up the config.
